src/client/modules/github/github.html

```diff
@@ -1,28 +1,50 @@
 <template name="github">
-  <h2>{{orgName}} contributions</h2>
-
-  Languages:
-  <ul>
-  {{#each languages}}
-    <li>{{language}}: {{percentage}}%</li>
-  {{/each}}
-  </ul>
-
-  Total: {{totalCount}}
-  <ul>
-  {{#each userData}}
-  <li>
-    <a href="{{member.html_url}}">
-      <div>{{member.login}}</div>
-      <div><img src="{{member.avatar_url}}&s=84"></div>
-    </a>
-    Total: {{count}}
-    <ul>
-    {{#each contributions}}
-      <li>Type: {{type}} / Repo: <a href="http://github.com/{{repo.name}}">{{repo.name}}</a></li>
-    {{/each}}
-    </ul>
-  </li>
-  {{/each}}
-  </ul>
+	<div class="col-lg-4">
+		<div class="org-sidebar-details jumbotron">
+			{{#each orgData}}
```

Comment:
I've been trying to use Adobe Brackets. Clearly I needed to adjust space/tab settings. Thanks for for the extra set of eyes!